.github/workflows/docs-pr.yml

```diff
@@ -24,11 +24,11 @@ jobs:
       - name: 'Set up JDKs'
         uses: ./.github/actions/setup-build-env
         with:
-          additional-java-version: 21
+          additional-java-version: 22
       - name: 'Install GraphViz'
         run: sudo apt update && sudo apt install --yes graphviz
       - name: 'Build Docs'
-        run: ./gradlew --stacktrace asciidoctor javadoc "-Dvariant=4.0" "-DjavaVersion=21"
+        run: ./gradlew --stacktrace asciidoctor javadoc "-Dvariant=4.0" "-DjavaVersion=22"
```

Comment:
I think we should keep this as 21 as we'll use 21 for the main docs run.

Reply:
Changed.